src/github/pullRequestManager.ts

```diff
@@ -98,6 +99,9 @@ export const onDidSubmitReview: vscode.Event<Comment[]> = _onDidSubmitReview.eve
 export class PullRequestManager {
 	static ID = 'PullRequestManager';
 	private _activePullRequest?: PullRequestModel;
+	public activeFileChanges?: GitFileChange[];
+	public activeOutdatedFileChanges?: GitFileChange[];
+	public activeComments?: Comment[];
 	private _credentialStore: CredentialStore;

```

Comment:
Not a fan of having view info in PullRequestManager. We can probably keep the information in `prsTreeDataProvider`. The `ReviewManager` takes care of git status change, and modifying `PullRequestManager.activePullRequest`, `prsTreeDataProvider` then listens to `activePullRequest` change event and reveal/refresh the tree when necessary.